.github/workflows/nix.yaml

```diff
@@ -131,3 +131,44 @@ jobs:
           name: debian
           path: "./packages/debian/*.deb"
           if-no-files-found: ignore
+      - name: Upload macOS Package to Github
+        if: steps.cfg.outputs.upload-to-github == 'true'
+        uses: actions/upload-artifact@v4
+        with:
+          name: macOS
+          path: "./packages/macOS/*.pkg"
+          if-no-files-found: ignore
+
+  install:
+    needs:
+      - build
+    if: ${{ contains(fromJSON(inputs.config).editions[0].gha-runner-labels, 'ubuntu-latest') }}
+    name: Install
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Is `fetch-depth` of 0, i.e. the entire history, needed here?

Reply:
No, I thought this was needed to fetch only 1 commit...